ibis/expr/operations/generic.py

```diff
@@ -295,14 +308,14 @@ class SearchedCase(Value):
 
     def __init__(self, cases, results, default):
         assert len(cases) == len(results)
```

Comment:
Why are we allowing empty `cases` and `results`?

Reply:
Lets keep the disussion of this to the other thread in here

Reply:
we decided in https://github.com/ibis-project/ibis/pull/9559 to disallow empty cases